pkg/restapi/issuer/operation/operations_test.go

```diff
@@ -42,14 +42,19 @@ func TestNew(t *testing.T) {
 
 func TestConnectWallet(t *testing.T) {
 	t.Run("test connect wallet - success", func(t *testing.T) {
-		c, err := New(&Config{AriesCtx: getAriesCtx()})
+		uiEndpoint := "/ui"
```

Comment:
note: this assumes there won't be a "combined" mode

Reply:
Yeah, we don' support combined mode (there is a separate vue/dockerfile for issuer and rp).